### PR TITLE
Properties&Confluence: translate 

### PR DESCRIPTION
**Pull request by starxingchenc**

### CODE REVIEW

File: src/plfa/part2/Confluence.lagda.md [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+<!--
 In this chapter we prove that beta reduction is _confluent_, a
 property also known as _Church-Rosser_. That is, if there are
 reduction sequences from any term `L` to two different terms `M₁` and
 `M₂`, then there exist reduction sequences from those two terms to
 some common term `N`. In pictures:
+-->
+
+在这一章我们将证明 β-规约是**合流的（Confluent）**，
+该性质同样以 *Church-Rosser* 闻名。也就是说，
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: 有点奇怪，好像应该改成 “该性质也被称为……”

File: src/plfa/part2/Confluence.lagda.md [hunk 2]
```diff
@@ -401,19 +629,39 @@ sub-par : ∀{Γ A B} {N N′ : Γ , A ⊢ B} {M M′ : Γ ⊢ A}
...
 so that every possible pair gives rise to a witness `L` given by
 performing enough beta reductions in parallel.
+-->
+
+合流性证明的核心是石头制成的，更确切地说，是钻石！
+【译注：在英文中 diamond 一词既指钻石，又指菱形。】
+我们将证明平行规约满足菱形性质，即若有 `M ⇛ N` 和 `M ⇛ N′`，
+那么对某个 `L` 有 `N ⇛ L` 和 `N′ ⇛ L`。
+典型的证明通过对 `M ⇛ N` 和 `M ⇛ N′` 归纳来完成，
+因此每一个可能的对都会在执行足够多次平行 β-规约后产生一个见证 `L`。
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: "witness"

File: src/plfa/part2/Confluence.lagda.md [hunk 3]
```diff
@@ -497,25 +782,52 @@ par-diamond : ∀{Γ A} {M N N′ : Γ ⊢ A}
...
 * Draw pictures that represent the proofs of each of the six cases in
   the direct proof of `par-diamond`. The pictures should consist of nodes
   and directed edges, where each node is labeled with a term and each
   edge represents parallel reduction.
+-->
+
+* 通过对 `M ⇛ N` 和 `M ⇛ N′` 归纳直接证明菱形性质 `par-diamond`。
+
+* 作图表示 `par-diamond` 的直接证明中的六种情况。
+  图应当包含节点和有向边，其中节点用项标号，边代表平行规约。
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: “标号”

File: src/plfa/part2/Confluence.lagda.md [hunk 4]
```diff
@@ -620,10 +969,25 @@ by Takahashi (1995) and Pfenning's 1992 technical report about the
...
+-->
+
+总而言之，这种基于平行规约的合流性的证明
+归功于 W. Tait 和 P. Martin-Löf （参见 Barendregt 1984，章节 3.2）。
+机械化的细节有多个来源。
+`subst-par` 引理是 Shafer、Tebbi 和 Smolka (ITP 2015) 的 “强替换性（Strong Substitutivity）” 引理。
+`par-triangle`、`strip` 和 `par-confluence` 的证明是基于 
+Takahashi (1995) 的 complete development 
+和 Pfenning 1992 年关于 Church-Rosser 定理的技术报告。
+此外，我们在 Isabelle 中咨询了 Nipkow 和 Berghofer 的机械化，
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: Isabelle好像是个proof assistant

应为“……在 Isabelle 中的机械化”

File: src/plfa/part2/Properties.lagda.md [hunk 5]
```diff
@@ -36,37 +47,80 @@ open import plfa.part2.Lambda
 ```
 
 
+<!--
 ## Introduction
+-->
+
+## 导言
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 在别的章中翻成了“简介”
不确定哪个更好

File: src/plfa/part2/Properties.lagda.md [hunk 6]
```diff
@@ -36,37 +47,80 @@ open import plfa.part2.Lambda
...
+归约为另一个项和良类型的项的概念。
 
+<!--
 Ultimately, we would like to show that we can keep reducing a term
 until we reach a value.  For instance, in the last chapter we showed
 that two plus two is four,
+-->
+
+最终，我们将要展示我们能够通过持续地对一个项做归约，直到它达到一个值。
+例如，在上一章中我们展示了二加上二的和是四，
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 似乎不太通顺
“二加二的和”？

File: src/plfa/part2/Properties.lagda.md [hunk 7]
```diff
@@ -36,37 +47,80 @@ open import plfa.part2.Lambda
...
+<!--
 So, either we have a value, and we are done, or we can take a reduction step.
 In the latter case, we would like to apply progress again. But to do so we need
 to know that the term yielded by the reduction is itself closed and well typed.
 It turns out that this property holds whenever we start with a closed,
 well-typed term.
+-->
 
+所以要么我们有一个值，这时我们已经完成了规约；要么我们可以进行一步规约。
+当处于后者的情况时，我们可以再一次应用进行性。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: "would like"
想要

File: src/plfa/part2/Properties.lagda.md [hunk 8]
```diff
@@ -85,11 +151,25 @@ treating a context as a function from identifiers to types —
 permits a simpler development. In particular, we can prove substitution preserves
 types without needing to develop a separate inductive definition of the
 `appears_free_in` relation.)
+-->
 
+（这一章启发自《软件基础》（_Software Foundations_）/《程序语言基础》（_Programming Language Foundations_）中对应的 _StlcProp_ 的内容。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: Chapter
_StlcProp_ 章节

File: src/plfa/part2/Properties.lagda.md [hunk 9]
```diff
@@ -99,41 +179,81 @@ V¬—→ V-ƛ        ()
...
 Well-typed values must take one of a small number of _canonical forms_,
 which provide an analogue of the `Value` relation that relates values
 to their types.  A lambda expression must have a function type,
 and a zero or successor expression must be a natural.
 Further, the body of a function must be well typed in a context
 containing only its bound variable, and the argument of successor
 must itself be canonical:
+-->
+
+良型的式子都属于少数几种**标准式（Canonical Form）**中的一种。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 统一为“良类型”？

File: src/plfa/part2/Properties.lagda.md [hunk 10]
```diff
@@ -99,41 +179,81 @@ V¬—→ V-ƛ        ()
...
+标准式提供了一种类似于 `Value` 的关系，关联值和它们所属的类型。
+一个 λ-表达式一定属于函数类型，同时零和后继表达式都属于自然数。
+更进一步说，此时函数的函数体必须在只包含它的约束变量的上下文中是良型的，
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 同上

File: src/plfa/part2/Properties.lagda.md [hunk 11]
```diff
@@ -242,21 +426,49 @@ Let's unpack the first three cases:
...
+    - 如果这个项还能够进行一步规约，我们就有了 `M —→ M′` 的论据，再由 `ξ-·₂`，
+      可知原来的项进行到 `L′ · M`。要应用规约步骤 `ξ-·₂` 需要我们提供项 `L` 是
+      一个值的论据，而之前对子项进行性的分析已经提供了需要的证明。
+
+    <!--
     - If the term is done, we have evidence that `M` is
       a value, so our original term steps by `β-ƛ`.
+    -->
+
+    - 如果这个项的规约已经完成了，我们便有了项 `M` 是值的论据。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 如果这个项的规约结束了

File: src/plfa/part2/Properties.lagda.md [hunk 12]
```diff
@@ -242,21 +426,49 @@ Let's unpack the first three cases:
...
+      我们项 `L` 是一个良类型的值的论据应用标准式引理
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 需要翻译

File: src/plfa/part2/Properties.lagda.md [hunk 13]
```diff
@@ -242,21 +426,49 @@ Let's unpack the first three cases:
...
 
+<!--
 The remaining cases are similar.  If by induction we have a
 `step` case we apply a `ξ` rule, and if we have a `done` case
 then either we have a value or apply a `β` rule.  For fixpoint,
 no induction is required as the `β` rule applies immediately.
+-->
 
+剩下的情况都很类似。如果我们由归纳得到了一个可以继续进行规约的
+情况 `case` 则应用一条 `ξ` 规则；如果得到的是已经完成规约的
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 情况 `step`

File: src/plfa/part2/Properties.lagda.md [hunk 14]
```diff
@@ -265,136 +477,306 @@ in full. In general, the rule of thumb is to consider the easy case
...
 implicit and so must be written out in full.  In the case for `β-ƛ`
 this requires that we match against the lambda expression `L` to
 determine its bound variable and body, `ƛ x ⇒ N`, so we can show that
 `L · M` reduces to `N [ x := M ]`.
+-->
 
+但这会导致证明变得不那么明晰易懂。
+比起有助于记忆的 `done` 与 `step`，现在我们只能用 `inj₁` 和 `inj₂`；
+同时项 `N` 也不再是隐式的，从而我们需要将其完整写出。
+当遇到 `β-ƛ` 的情况时，需要我们对 λ-表达式做匹配，以决定它的约束变量和函数体，
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: lambda expression `L`

File: src/plfa/part2/Properties.lagda.md [hunk 15]
```diff
@@ -433,34 +841,74 @@ rename ρ (⊢suc ⊢M)         =  ⊢suc (rename ρ ⊢M)
...
 ```
+
+<!--
 As before, let `ρ` be the name of the map that takes evidence that
 `x` appears in `Γ` to evidence that `x` appears in `Δ`.  We induct
 on the evidence that `M` is well typed in `Γ`.  Let's unpack the
 first three cases:
+-->
 
+像之前一样，令 `ρ` 为变量 `x` 出现在上下文 `Γ` 中的论据至 `x` 出现在 `Δ` 中变量的映射。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 令 `ρ` 为 “变量 `x` 出现在上下文 `Γ` 中的论据” 至 “变量 `x` 出现在 `Δ` 的论据” 的映射。

File: src/plfa/part2/Properties.lagda.md [hunk 16]
```diff
@@ -494,14 +952,27 @@ drop {Γ} {x} {M} {A} {B} {C} ⊢M = rename ρ ⊢M
...
+<!--
 Here map `ρ` can never be invoked on the inner occurrence of `x` since
 it is masked by the outer occurrence.  Skipping over the `x` in the
 first position can only happen if the variable looked for differs from
 `x` (the evidence for which is `x≢x` or `z≢x`) but if the variable is
 found in the second position, which also contains `x`, this leads to a
 contradiction (evidenced by `x≢x refl`).
+-->
+
+在这里映射 `ρ` 不可能调用较里出现的 `x`，由于它被较外的出现遮盖了。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 被调用

File: src/plfa/part2/Properties.lagda.md [hunk 17]
```diff
@@ -494,14 +952,27 @@ drop {Γ} {x} {M} {A} {B} {C} ⊢M = rename ρ ⊢M
...
+忽略掉在首位的 `x` 的情况只可能发生在当前寻找的变量不同于 `x` 的情
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 的情况重复了

File: src/plfa/part2/Properties.lagda.md [hunk 18]
```diff
@@ -518,32 +989,68 @@ swap {Γ} {x} {y} {M} {A} {B} {C} x≢y ⊢M = rename ρ ⊢M
...
+
+## 替换
 
+<!--
 The key to preservation – and the trickiest bit of the proof – is
 the lemma establishing that substitution preserves types.
+-->
 
+证明保型性的关键——也正是证明最具有技巧性的部分——是
+一条证明替换保持赋型的引理。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md:1017]: 去掉“一条”？

File: src/plfa/part2/Properties.lagda.md [hunk 19]
```diff
@@ -518,32 +989,68 @@ swap {Γ} {x} {y} {M} {A} {B} {C} x≢y ⊢M = rename ρ ⊢M
...
 free variable (the bound variable of the lambda abstraction, or
 similarly for case or fixpoint). However, substitution
 is defined by recursion, and as we descend into terms with bound
 variables the context grows.  So for the induction to go through,
 we require an arbitrary context `Γ`, as in the statement of the lemma.
+-->
 
+我们所关注的是规约闭项，这意味着每当我们应用 `β`-规约时，
+所替换的项只含有一个自由变量（也就是所对应 λ-抽象、
+对自然数分项或不动点表达式的绑定变量）。然而，替换是通过
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md:1043]: 同上 对自然数分项

File: src/plfa/part2/Properties.lagda.md [hunk 20]
```diff
@@ -569,9 +1076,15 @@ subst {x = y} ⊢V (⊢μ {x = x} ⊢M) with x ≟ y
 ... | yes refl        =  ⊢μ (drop ⊢M)
 ... | no  x≢y         =  ⊢μ (subst ⊢V (swap x≢y ⊢M))
 ```
+
+<!--
 We induct on the evidence that `N` is well typed in the
 context `Γ` extended by `x`.
+-->
+
+我们对 `N` 在由 `x`拓展 `Γ` 后得到的上下文中是良赋型的论据做归纳。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 空格

File: src/plfa/part2/Properties.lagda.md [hunk 21]
```diff
@@ -584,21 +1097,47 @@ Using the name `y` here is consistent with the naming in the original
 definition of substitution in the previous chapter.  The proof never
 mentions the types of `x`, `y`, `V`, or `N`, so in what follows we
 choose type names as convenient.
-
+-->
+
+首先我们注意到一个有关命名的问题。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: "wee" “微小的问题”

File: src/plfa/part2/Properties.lagda.md [hunk 22]
```diff
@@ -584,21 +1097,47 @@ Using the name `y` here is consistent with the naming in the original
...
+在引理的陈述中，被替换的变量 `x` 是一个隐式参数，同时在变量、抽象、
+对自然数分项和不动点的赋型规则中，相关变量 `x` 是一个隐式参数，因此
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md:1104]: 同上

Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: [cd18961](https://github.com/starxingchenc/PLFA-zh/commit/cd18961ab5d9414b13c88344c3ce875c2fa30773)

File: src/plfa/part2/Properties.lagda.md [hunk 23]
```diff
@@ -584,21 +1097,47 @@ Using the name `y` here is consistent with the naming in the original
...
+我们使用形同 `{x = y}` 的语法来将 `y` 与被替换的变量绑定，
+用 `{x = x}` 来将 `x` 与 `` ⊢` ``、`⊢ƛ`、`⊢case` 和 `⊢μ` 中相关的变量绑定。
+在这里用 `y` 这一名字业余前一章中替换的原始定义一致。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 也与

File: src/plfa/part2/Properties.lagda.md [hunk 24]
```diff
@@ -717,36 +1256,74 @@ Now that naming is resolved, let's unpack the first three cases:
   Applying the induction hypothesis for `L` and `M` and the typing
   rule for applications yields the required conclusion.
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md:1257]: 需要翻译

File: src/plfa/part2/Properties.lagda.md [hunk 25]
```diff
@@ -584,21 +1097,47 @@ Using the name `y` here is consistent with the naming in the original
...
+-->
+
+首先我们注意到一个有关命名的问题。
+在引理的陈述中，被替换的变量 `x` 是一个隐式参数，同时在变量、抽象、
+对自然数分项和不动点的赋型规则中，相关变量 `x` 是一个隐式参数，因此
+我们使用形同 `{x = y}` 的语法来将 `y` 与被替换的变量绑定，
+用 `{x = x}` 来将 `x` 与 `` ⊢` ``、`⊢ƛ`、`⊢case` 和 `⊢μ` 中相关的变量绑定。
+在这里用 `y` 这一名字业余前一章中替换的原始定义一致。
+在证明中从来没有提及过 `x`、`y`、`V` 或 `N` 的类型，所以接下来我们按照惯例
+选取类型的名字：
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 因此在下面的内容中，我们尽可能方便地选择类型名称。

File: src/plfa/part2/Properties.lagda.md [hunk 26]
```diff
@@ -910,44 +1613,89 @@ eval {L} (gas (suc m)) ⊢L with progress ⊢L
...
 ```
+
+<!--
 Let `L` be the name of the term we are reducing, and `⊢L` be the
 evidence that `L` is well typed.  We consider the amount of gas
 remaining.  There are two possibilities:
+-->
 
+令 `L` 是我们要规约的项的名称，`⊢L` 是项 `L` 是良类型的论据。
+我们考虑剩余的燃料量。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: There are two possibilities
未翻译

File: src/plfa/part2/Confluence.lagda.md [hunk 27]
```diff
@@ -169,9 +270,14 @@ beta-par {Γ} {★} {(ƛ N) · M} β = pbeta par-refl par-refl
 beta-par {Γ} {★} {ƛ N} (ζ r) = pabs (beta-par r)
 ```
 
+<!--
 With this lemma in hand we complete the only-if direction,
 that `M —↠ N` implies `M ⇛* N`. The proof is a straightforward
 induction on the reduction sequence `M —↠ N`.
+-->
+
+证明了该引理后我们便可完成证明，
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: only-if direction

File: src/plfa/part2/Confluence.lagda.md [hunk 28]
```diff
@@ -183,10 +289,17 @@ betas-pars {Γ} {A} {.L} {N} (L —→⟨ b ⟩ bs) =
...
 
+<!--
 Now for the other direction, that `M ⇛* N` implies `M —↠ N`.  The
 proof of this direction is a bit different because it's not the case
 that `M ⇛ N` implies `M —→ N`. After all, `M ⇛ N` performs many
 reductions. So instead we shall prove that `M ⇛ N` implies `M —↠ N`.
+-->
+
+现在考虑命题的充分性，即 `M ⇛* N` 蕴含 `M —↠ N`。
+该方向的证明有一点不同，因为它不是 `M ⇛ N` 蕴含 `M —→ N` 的情形。
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: 充分性的证明

File: src/plfa/part2/Properties.lagda.md [hunk 29]
```diff
@@ -1221,30 +1999,68 @@ _ : eval (gas 100) ⊢2+2ᶜ ≡
...
 #### Exercise `mul-eval` (recommended)
+-->
 
+#### 练习 `mul-eval`（推荐）
+
+<!--
 Using the evaluator, confirm that two times two is four.
+-->
+
+用这个求值器来验证二乘以二的积是四。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 二乘二

File: src/plfa/part2/Properties.lagda.md [hunk 30]
```diff
@@ -1253,27 +2069,60 @@ Its opposite is _subject expansion_, which holds if
 `M —→ N` and `∅ ⊢ N ⦂ A` imply `∅ ⊢ M ⦂ A`.
 Find two counter-examples to subject expansion, one
 with case expressions and one not involving case expressions.
+-->
 
+我们说 `M` **规约**至 `N` 如果 `M —→ N`，
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 声称

File: src/plfa/part2/Properties.lagda.md [hunk 31]
```diff
@@ -1253,27 +2069,60 @@ Its opposite is _subject expansion_, which holds if
...
+但是我们可以通过说 `N` **扩展（Expand）**至 `M` 来描述相同的情形。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 同上

File: src/plfa/part2/Properties.lagda.md [hunk 32]
```diff
@@ -1253,27 +2069,60 @@ Its opposite is _subject expansion_, which holds if
...
+保型性有时也被叫做**子规约（Subject Reduction）**。
+它的对应是**子扩展（Subject Expansion）**，
+如果 `M —→ N` 和 `∅ ⊢ N ⦂ A` 蕴含 `∅ ⊢ M ⦂ A`。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md:2078]: 子扩展成立，……

File: src/plfa/part2/Properties.lagda.md [hunk 33]
```diff
@@ -1253,27 +2069,60 @@ Its opposite is _subject expansion_, which holds if
...
+找到两个子扩展的反例，一个涉及 case 表达式而另一个不涉及。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: case

File: src/plfa/part2/Properties.lagda.md [hunk 34]
```diff
@@ -1413,107 +2365,225 @@ and add
...
+-->
+
+上述证明中的 18 行中有 5 行是多余的，
+例如，当一个规则是 `ξ-·₁` 而另一个是 `ξ-·₂` 的情况被考虑两次，
+一次是先有 `ξ-·₁`，然后有 `ξ-·₂`，另一次将两者互换。
+我们可能想做的是删除多余的行并在证明的底部添加
+
+    det M—→M′ M—→M″ = sym (det M—→M″ M—→M′)
 
+但这不起作用：终结检查器报错，因为参数只是切换了顺序，而且没有任何一个变得更小。
```
Comment (starxingchenc) [inline src/plfa/part2/Properties.lagda.md]: 交换了顺序

File: src/plfa/part2/Confluence.lagda.md [hunk 35]
```diff
@@ -316,19 +495,30 @@ par-subst-exts s {x = Z} = pvar
...
 
+<!--
 The next lemma that we need for proving that substitution respects
 parallel reduction is the following which states that
 simultaneous substitution commutes with single substitution. We import this
 lemma from Chapter [Substitution](/Substitution/)
 and restate it below.
+-->
+
+下一个我们需要证明的引理是替换遵从平行规约关系，如下文所示，
```
Comment (starxingchenc) [inline src/plfa/part2/Confluence.lagda.md]: 为了证明替换遵从平行规约关系，我们需要证明的下一个引理……

**Comment by alissa-tung** [comment]
I gonna commit a list of commits to apply changes. After resolve those conversations all of them would be squashed.

File: src/plfa/part2/Confluence.lagda.md (hunk 1 above)
Comment (alissa-tung) [reply src/plfa/part2/Confluence.lagda.md]: thx
0fad44a

File: src/plfa/part2/Confluence.lagda.md (hunk 35 above)
Comment (alissa-tung) [reply src/plfa/part2/Confluence.lagda.md]: thx
deeb0e2

File: src/plfa/part2/Properties.lagda.md (hunk 34 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: thx
[95a4482](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/95a4482e75d63ef1921a1a214c9edcf9bfad56de)

File: src/plfa/part2/Properties.lagda.md (hunk 33 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: thx
[875d053](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/875d0534424ecc13577d9ecf506d90307554d7e9)

**Comment by alissa-tung** [comment]
Disable CI temporarily, we can revert and squash this commit [344508c](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/344508ce843b560faefa7c9867432b421f4af392) after all work are done.

File: src/plfa/part2/Properties.lagda.md (hunk 30 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: [d4d7638](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/d4d7638bcbc29295655fa6036895b405f739c180)

File: src/plfa/part2/Properties.lagda.md (hunk 31 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: [d4d7638](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/d4d7638bcbc29295655fa6036895b405f739c180)

File: src/plfa/part2/Properties.lagda.md (hunk 29 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: [dba1576](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/dba1576263192581ad3e72728426851aed1f3144)

File: src/plfa/part2/Properties.lagda.md (hunk 6 above)
Comment (alissa-tung) [reply src/plfa/part2/Properties.lagda.md]: f57721b

File: src/plfa/part2/Confluence.lagda.md (hunk 27 above)
Comment (starxingchenc) [reply src/plfa/part2/Confluence.lagda.md]: [b10e31d](https://github.com/Agda-zh/PLFA-zh/pull/94/commits/b10e31d3cb581a2427a97b4efe4e42e560edbda0)

File: src/plfa/part2/Confluence.lagda.md (hunk 28 above)
Comment (starxingchenc) [reply src/plfa/part2/Confluence.lagda.md]: [020ece7](https://github.com/starxingchenc/PLFA-zh/commit/020ece745227a93d810217411780f0b94f012856)

File: src/plfa/part2/Properties.lagda.md (hunk 25 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [32d4fa8](https://github.com/starxingchenc/PLFA-zh/commit/32d4fa8f756148238aead5abef3cf632a3335281)

File: src/plfa/part2/Properties.lagda.md (hunk 7 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [5673fa4](https://github.com/starxingchenc/PLFA-zh/commit/5673fa47e24bd752aef28d5a22c61b8f03c73ada)

File: src/plfa/part2/Properties.lagda.md (hunk 8 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [1ea8f00](https://github.com/starxingchenc/PLFA-zh/commit/1ea8f00c38ecee90d1d14de02985267f0aa45226)

File: src/plfa/part2/Properties.lagda.md (hunk 9 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [25fd9c4](https://github.com/starxingchenc/PLFA-zh/commit/25fd9c4784661cd3a14fc3eaf74a8acdfe856db5)

File: src/plfa/part2/Properties.lagda.md (hunk 10 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [25fd9c4](https://github.com/starxingchenc/PLFA-zh/commit/25fd9c4784661cd3a14fc3eaf74a8acdfe856db5)

File: src/plfa/part2/Properties.lagda.md (hunk 13 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [4432bae](https://github.com/starxingchenc/PLFA-zh/commit/4432bae392684e313f9ed7f773a668dfcfbc2f63)

File: src/plfa/part2/Properties.lagda.md (hunk 14 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [6c94e5a](https://github.com/starxingchenc/PLFA-zh/commit/6c94e5a7d8e89a0ac96615d9362b46420dfe2769)

File: src/plfa/part2/Properties.lagda.md (hunk 20 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [f7acb62](https://github.com/starxingchenc/PLFA-zh/commit/f7acb62d4460cbdc07423728f61b70fd5dc63ae7)

File: src/plfa/part2/Properties.lagda.md (hunk 21 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [c48c668](https://github.com/starxingchenc/PLFA-zh/commit/c48c668d28f0ef690335a54b085ffcdb1d2295cc)

File: src/plfa/part2/Properties.lagda.md (hunk 23 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [f633ab8](https://github.com/starxingchenc/PLFA-zh/commit/f633ab8f22dfa93f97ba4625dec8e8740008bbc9)

File: src/plfa/part2/Confluence.lagda.md (hunk 2 above)
Comment (starxingchenc) [reply src/plfa/part2/Confluence.lagda.md]: 改为“证明”
[3c9fe22](https://github.com/starxingchenc/PLFA-zh/commit/3c9fe222f175b81d98b910ffe9d665f78562fb6d)

File: src/plfa/part2/Confluence.lagda.md (hunk 3 above)
Comment (starxingchenc) [reply src/plfa/part2/Confluence.lagda.md]: 改为“标记”
[57d134e](https://github.com/starxingchenc/PLFA-zh/commit/57d134e02ac10d4bbc837e757135d24c3be4b2b3)

File: src/plfa/part2/Confluence.lagda.md (hunk 4 above)
Comment (starxingchenc) [reply src/plfa/part2/Confluence.lagda.md]: [937dd55](https://github.com/starxingchenc/PLFA-zh/commit/937dd55401051b43011aab5cdfee3b5da8d4a117)

File: src/plfa/part2/Properties.lagda.md (hunk 5 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: 暂时修改为“简介”

File: src/plfa/part2/Properties.lagda.md (hunk 11 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [9f5e790](https://github.com/starxingchenc/PLFA-zh/commit/9f5e7904b8ed8d3c5109db709b2aefe97aaa4fad)

File: src/plfa/part2/Properties.lagda.md (hunk 12 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [175e369](https://github.com/starxingchenc/PLFA-zh/commit/175e36903902855cdba0b9cbc8abde467c6fbf64url)

File: src/plfa/part2/Properties.lagda.md (hunk 15 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [e746663](https://github.com/starxingchenc/PLFA-zh/commit/e746663ff2d60cc323dcd0cf89a345795553bc9a)

File: src/plfa/part2/Properties.lagda.md (hunk 16 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [da0cfce](https://github.com/starxingchenc/PLFA-zh/commit/da0cfce622b0b7066b07c7f23ea035540b81e760)

File: src/plfa/part2/Properties.lagda.md (hunk 17 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [bbef5dd](https://github.com/starxingchenc/PLFA-zh/commit/bbef5dd797fd73f87cbf9a1aef1c244648b609c4)

File: src/plfa/part2/Properties.lagda.md (hunk 26 above)
Comment (starxingchenc) [reply src/plfa/part2/Properties.lagda.md]: [8cdc157](https://github.com/starxingchenc/PLFA-zh/commit/8cdc15745c84a2e4cbf8d37ca23195c77e78f2b0)

**Pull request closed without merging**